### ISSUE TITLE
Custom dataset and pretrained weights

### ISSUE THREAD

**Issue by MaleshwarSastri** [cyvius96/few-shot-meta-baseline#5]
HI,
im a newbie to deep learning.i want to train my own dataset and evaluate it.Can u elaborate on how it can be done and how do i pass the model path,my query and target images to the evaluation script?

**Comment by cyvius96** [comment]
Hi, you can follow these steps:
1. Create your custom dataset in datasets folder (you may take mini_imagenet as a reference)
2. Create your custom config and write down the args for building the dataset (you may take configs/train_classifier_mini.yaml as a reference)
3. Run the code

If you have the dataset with image folders, it should be more convenient that you can reuse ImageFolder class (see train_classifier_im800.yaml for example), which requires a json file that defines the base classes and novel classes.

The code is sampling tasks during testing, if you want to test a specific task, you may need to feed them directly in the code.

I guess these steps would be a bit challenging for a beginner. Good luck!

**Comment by MaleshwarSastri** [comment]
Thanks for the instant reply will try it out.I only have a single 12gb GPU will that be sufficient for training the model.

**Comment by cyvius96** [comment]
You can try to run the code on your single GPU.
If the memory is not enough, you can try to reduce batch_size/ep_per_batch in configs, or to add gradient accumulation in the code.
